src/index.js

```diff
@@ -2409,6 +2409,9 @@ const createScatterplot = (
    * @returns {Promise<void>}
    */
   const zoomToPoints = (pointIdxs, options = {}) => {
+    if(!searchIndex){
+      return false;
+    }
```

Comment:
```suggestion
    if (!isInit) return Promise.reject(new Error(ERROR_NOT_INITIALIZED))
```

Can you add the following line to the end of the `constants.js` file and import it?

```js
export const ERROR_NOT_INITIALIZED = 'Points have not been initialized yet';
```